### PR TITLE
Fix environment variables, clever truncation on /, and allow ragged when wider than column

### PR DESCRIPTION
**Pull request by dchw**
Some sample for how it will look with these changes:

```
           buildkitd | Found buildkit daemon as docker container (earthly-buildkitd)
           buildkitd | Newer image available. Restarting buildkit daemon...
           buildkitd | ...Done
g/e/hello-world:master+base | --> FROM busybox:1.32.0
               +base | --> FROM golang:1.13-alpine3.11
             context | --> local context ./examples/go
g/e/hello-world:master+base | resolve docker.io/library/busybox:1.32.0@sha256:a9286defaba7b3a519d585ba0e37d0b2cbee74ebfe590960b0b1d6a5e97d1e1d 100%
               +base | resolve docker.io/library/golang:1.13-alpine3.11@sha256:ec6dcf15073c307fbcfc3149efe8835f3ec2bd0a0cb49aaaee4949cfc4c86b65 100%
             context | transferring ./examples/go: 100%
               +base | *cached* --> RUN apk add --update --no-cache bash bash-completion binutils ca-certificates coreutils curl findutils g++ git grep less make openssl shellcheck util-linux
               +base | *cached* --> WORKDIR /earthly
  ./examples/go+base | *cached* --> WORKDIR /go-example
  ./examples/go+deps | *cached* --> COPY go.mod go.sum ./
  ./examples/go+deps | *cached* --> RUN go mod download
 ./examples/go+build | *cached* --> COPY main.go .
 ./examples/go+build | *cached* --> RUN go build -o build/go-example main.go
 ./examples/go+build | *cached* --> SAVE ARTIFACT build/go-example ./examples/go+build/go-example
./examples/go+docker | *cached* --> COPY ./examples/go+build/go-example .
g/e/hello-world:master+hello | *cached* --> RUN echo 'Hello, world!'
           +examples | Target github.com/earthly/earthly:corey/logging-alignment-truncation+examples built successfully
=========================== SUCCESS ===========================
Loaded image: go-example:latest
./examples/go+docker | Image github.com/earthly/earthly/examples/go:corey/logging-alignment-truncation+docker as go-example:latest
 ./examples/go+build | *cached* --> SAVE ARTIFACT build/go-example ./examples/go+build/go-example AS LOCAL build/go-example
 ./examples/go+build | Artifact github.com/earthly/earthly/examples/go:corey/logging-alignment-truncation+build/go-example as local build/go-example
  ./examples/go+deps | *cached* --> SAVE ARTIFACT go.mod ./examples/go+deps/go.mod AS LOCAL go.mod
  ./examples/go+deps | Artifact github.com/earthly/earthly/examples/go:corey/logging-alignment-truncation+deps/go.mod as local go.mod
  ./examples/go+deps | *cached* --> SAVE ARTIFACT go.sum ./examples/go+deps/go.sum AS LOCAL go.sum
  ./examples/go+deps | Artifact github.com/earthly/earthly/examples/go:corey/logging-alignment-truncation+deps/go.sum as local go.sum

```

### CODE REVIEW

**Review by vladaionescu — APPROVED** [review]
Nice! 🎉

**Comment by dchw** [comment]
See #445 for details regarding these changes.

**Review by alexcb — APPROVED** [review]

File: conslogging/conslogging.go [hunk 1]
```diff
@@ -239,12 +239,24 @@ func (cl ConsoleLogger) prettyPrefix() string {
...
+	if len(cl.prefix) > cl.prefixPadding {
+		parts := strings.Split(cl.prefix, "/")
+		target := parts[len(parts)-1]
+
+		truncated := ""
+		for _, part := range parts[:len(parts)-1] {
+			letter := ""
+			if len(part) > 0 {
+				letter = string(part[0])
+			}
```
Comment (vladaionescu) [inline conslogging/conslogging.go:253]: I just realized - we should avoid truncating `..` to `.`

**Review by vladaionescu — CHANGES_REQUESTED** [review]

File: conslogging/conslogging.go (hunk 1 above)
Comment (dchw) [reply conslogging/conslogging.go:253]: Hm interesting; I didnt realize we could have relative paths? What if we ran them through https://golang.org/pkg/path/#Clean before truncation, this wouldn't be an issue then?

Comment (vladaionescu) [reply conslogging/conslogging.go:253]: This is for the local target case, not remote. For example this `FROM` here https://github.com/earthly/earthly/blob/master/examples/tests/Earthfile#L1. It wouldn't work for that unfortunately.

**Review by vladaionescu — APPROVED** [review]

**Pull request merged**